TestApp/Model/Person.cs

```diff
@@ -0,0 +1,33 @@
+﻿using FluentValidation;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace TestApp.Model
+{
+    public class Person
+    {
+        public Person()
+        {
+            this.Identities = new HashSet<Identifier>();
+        }
+        [Key]
```

Comment:
its not required if the convention is followed

Reply:
Ok